### ISSUE TITLE
Implement autodetection of number of training examples

### ISSUE THREAD

**Issue by vigliensoni** [vigliensoni/R-VAE#7]
Since a minimum amount of training data is needed to learn something (i.e., to decrease the loss), a system to duplicate the number of given examples is needed. 

As of now, it is not clear what is the minimum amount of data needed. However, preliminary experiments showed that a decent model with only 10 clips was achieved by duplicating the data 64 times (i.e., 640 examples). This number could be less or more, but it can be used to provide the logic of the system.

**Comment by vigliensoni** [comment]
Should be handled here: 

https://github.com/vigliensoni/R-VAE/blob/d22363e6fb9b1abd913d6db2c5d47835be3f367a/rhythmvae.js#L143-L169

**Comment by vigliensoni** [comment]
Data duplication is only working when using folders (i.e., at the moment of selecting data), not right before training.
